tests/features/publication.feature

```diff
@@ -37,7 +103,16 @@ Feature: Publication content creation
 
   @javascript
   Scenario: Length limited fields are truncating characters exceeding the configured limit.
-    Given I am logged in as a user with the "create oe_publication content, access content, edit own oe_publication content, view published skos concept entities" permission
+    Given I am logged in as a user with the "create oe_publication content, access content, edit own oe_publication content, view published skos concept entities, manage corporate content entities" permission
+
+    And the following documents:
+    | name          | file       |
+    | My Document 1 | sample.pdf |
+
+    And the following images:
+    | name         | file           | alt          |
+    | Sample image | example_1.jpeg | example text |
```

Comment:
Thumbnail is not required and this scenario tests the maxlength limits. We don't need to create and use it.

Reply:
This is removed.